package.json

```diff
@@ -11,45 +11,53 @@
     "ckeditor5-math",
     "katex"
   ],
-  "resolutions": {
-    "postcss-loader": "^4.3.0",
-    "mini-css-extract-plugin": "2.4.2"
-  },
+  "main": "src/index.js",
   "dependencies": {
-    "@ckeditor/ckeditor5-clipboard": "35.2.1",
-    "@ckeditor/ckeditor5-core": "35.2.1",
-    "@ckeditor/ckeditor5-engine": "35.2.1",
-    "@ckeditor/ckeditor5-inspector": "3.0.0",
-    "@ckeditor/ckeditor5-ui": "35.2.1",
-    "@ckeditor/ckeditor5-undo": "35.2.1",
-    "@ckeditor/ckeditor5-utils": "35.2.1",
-    "@ckeditor/ckeditor5-widget": "35.2.1"
+    "@ckeditor/ckeditor5-clipboard": "40.0.0",
+    "@ckeditor/ckeditor5-core": "40.0.0",
+    "@ckeditor/ckeditor5-engine": "40.0.0",
+    "@ckeditor/ckeditor5-ui": "40.0.0",
+    "@ckeditor/ckeditor5-undo": "40.0.0",
+    "@ckeditor/ckeditor5-utils": "40.0.0",
+    "@ckeditor/ckeditor5-widget": "40.0.0",
+    "ckeditor5": "40.0.0"
   },
   "devDependencies": {
-    "@ckeditor/ckeditor5-basic-styles": "35.2.1",
-    "@ckeditor/ckeditor5-dev-tests": "31.1.1",
-    "@ckeditor/ckeditor5-editor-classic": "35.2.1",
-    "@ckeditor/ckeditor5-editor-inline": "35.2.1",
-    "@ckeditor/ckeditor5-essentials": "35.2.1",
-    "@ckeditor/ckeditor5-paragraph": "35.2.1",
-    "@ckeditor/ckeditor5-theme-lark": "35.2.1",
+    "@ckeditor/ckeditor5-autoformat": "40.0.0",
+    "@ckeditor/ckeditor5-basic-styles": "40.0.0",
+    "@ckeditor/ckeditor5-block-quote": "40.0.0",
+    "@ckeditor/ckeditor5-code-block": "40.0.0",
+    "@ckeditor/ckeditor5-core": "40.0.0",
+    "@ckeditor/ckeditor5-dev-tests": "39.2.1",
+    "@ckeditor/ckeditor5-dev-translations": "^32.1.2",
+    "@ckeditor/ckeditor5-editor-classic": "40.0.0",
+    "@ckeditor/ckeditor5-editor-inline": "40.0.0",
+    "@ckeditor/ckeditor5-essentials": "40.0.0",
+    "@ckeditor/ckeditor5-heading": "40.0.0",
+    "@ckeditor/ckeditor5-image": "40.0.0",
+    "@ckeditor/ckeditor5-indent": "40.0.0",
+    "@ckeditor/ckeditor5-inspector": "^4.1.0",
+    "@ckeditor/ckeditor5-link": "40.0.0",
+    "@ckeditor/ckeditor5-list": "40.0.0",
+    "@ckeditor/ckeditor5-media-embed": "40.0.0",
+    "@ckeditor/ckeditor5-package-tools": "^1.1.0",
+    "@ckeditor/ckeditor5-paragraph": "40.0.0",
+    "@ckeditor/ckeditor5-table": "40.0.0",
+    "@ckeditor/ckeditor5-theme-lark": "40.0.0",
+    "@ckeditor/ckeditor5-upload": "40.0.0",
     "css-loader": "^5.2.7",
-    "eslint": "^7.1.0",
-    "eslint-config-ckeditor5": "^3.1.1",
+    "eslint": "^7.32.0",
+    "eslint-config-ckeditor5": "^5.1.1",
     "html-webpack-plugin": "^5.3.1",
```

Comment:
it was removed in https://github.com/isaul32/ckeditor5-math/commit/383361733a258bdc0a2ba610bd8be7004a125744#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L39